camille/process/mooring_fatigue.py

```diff
@@ -20,17 +22,21 @@ def process(series, window_length=3600, fs=5):
     Returns
     -------
     pandas.Series
+        Series defining calculated damage. Original data index is
+        sampled as per bin size and set to bin left boundary
```

Comment:
I find this a bit confusing. 

Perhaps: Damage calculations for the given windows. Data is indexed using the window left boundary index.